scripts/gateway-docker-upgrade.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo2() {
+  echo $* >&2
+}
+
+CONTAINER_NAME=firezone-gateway
+TARGET_IMAGE="${TARGET_IMAGE:-us-east1-docker.pkg.dev/firezone-prod/firezone/gateway:1}"          # use DOCKER_IMAGE to override default image
+
+# find existing running gateway
+CURRENTLY_RUNNING=$(docker ps --format "{{.Names}} {{.Image}}" | grep -e "$CONTAINER_NAME" | awk '{print $1}')
```

Comment:
What would you think about looking for an image name rather than a container name?  Was just thinking that there may be a chance that someone runs their container as a different name than we'd expect, but it'd be much less likely they'd be running a custom image.

Reply:
Good call, updated